README.md

```diff
@@ -24,12 +23,22 @@ The source code is located in `./src/boostvqe/.` and its composed of:
 * `plotscripts.py`: plotting functions.
 * `compiling_XXZ.py`: compilation for XXZ model.
 
-## How to run the code
+## Example
 
-For further information about the inputs:
+It follows a python snippet explaining how to run the boosting
 
-```sh
-python main.py --help
+```py
+
+from boostvqe.boost import main
+from boostvqe.ansatze import hdw_efficient
+
+from qibo.models.dbi.double_bracket import DoubleBracketGeneratorType
+
+help(main) #If you need some info
```

Comment:
This is ok, however I don't like that when launching the script it will open a separate window that you need to close. Perhaps it is better just to drop it.